### ISSUE TITLE
Ctrl+S on an image corrupts the image

### ISSUE THREAD

**Issue by DuckyMacfly** [photopea/photopea#5151]
When I edit images I usually just press Ctrl+S to save any changes I've made. However, I can't anymore because anytime I do it just says that the image is now corrupted. Everything was fine yesterday but now it's not working for some reason. I'm talking about rewriting the PNG itself btw, not a PSD file. It also works if I just export the image, though it's annoying because I'm so used to Ctrl+S. 

Hopefully this gets fixed.

![image](https://user-images.githubusercontent.com/79030450/205459693-debaaf68-5bdb-4535-ae71-0811a5549b6b.png)
![image](https://user-images.githubusercontent.com/79030450/205459734-398980a6-22f9-4012-906b-64fae3fe89c7.png)

**Comment by photopea** [comment]
Try this, let me know if it helped https://www.photopea.com/tuts/photopea-doesnt-work-what-to-do/

**Comment by DuckyMacfly** [comment]
> Try this, let me know if it helped https://www.photopea.com/tuts/photopea-doesnt-work-what-to-do/

Ok, I tested out all the things you listed but sadly it doesn't seem to work still. It corrupts both JPG and PNG, my Google Chrome is up-to-date and it still doesn't work, even in Incognito Mode. I also tried Microsoft Edge, that didn't work as well, and I've done nothing that would render today broken but yesterday completely fine, so yeah, pretty weird.
